Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -608,8 +608,10 @@ public extension String {
     ///
     /// - Parameter range: Half-open range.
     subscript(safe range: CountableRange<Int>) -> String? {
-        guard let lowerIndex = index(startIndex, offsetBy: max(0, range.lowerBound), limitedBy: endIndex) else { return nil }
+        guard !isEmpty && range.lowerBound >= 0 else { return nil }
+        guard let lowerIndex = index(startIndex, offsetBy: range.lowerBound, limitedBy: endIndex) else { return nil }
         guard let upperIndex = index(lowerIndex, offsetBy: range.upperBound - range.lowerBound, limitedBy: endIndex) else { return nil }
```

Comment:
Combine the guards together. Do the same for the other function.
```suggestion
        guard !isEmpty,
            range.lowerBound >= 0,
            let lowerIndex = index(startIndex, offsetBy: range.lowerBound, limitedBy: endIndex),
            let upperIndex = index(lowerIndex, offsetBy: range.upperBound - range.lowerBound, limitedBy: endIndex) else {
                 return nil
        }
```

Reply:
Combined guard statements for both subscripts